### ISSUE TITLE
Retain identical pop up on hover behavior for words set to "well-known" as learning statuses 1-5

### ISSUE THREAD

**Issue by firion1234** [jzohrab/lute#18]
**Is your feature request related to a problem? Please describe.**

I believe that the current behavior discourages ever setting a word to well-known. My stated reasoning is as follows: A user may have intimate familiarity with the more common forms of a word and consequently wish to remove the colored highlight of learning statuses 1-5 but still desire to have quick reference to the parent word. The current behavior requires the user to completely open the entry for a word in order to access its definition, parent word, and tags when a mere hover should suffice. 

Take this use case as an example; in Latin, in order to be able to conjugate a given verb into every tense, voice, and mood, the learner is required to have mastered the verb's so called "principal parts". The more frequent a learner's exposure to the principal parts, the easier their ultimate acquisition will be. The parent word box is an excellent place to put not only the principal parts but also a short definition of the headword for quick reference.

**Describe the solution you'd like**

Ideally, Lute would retain the identical pop up on hover behavior for words set to "well-known" as it does for learning statuses 1-5.

**Comment by jzohrab** [comment]
Thanks for the note, yes, this makes sense.

I had originally hidden this pop-up since I often set words as "well-known" without adding any additional content.  The pop-up was kind of annoying in those cases.  Maybe in the (near?) future I'll add a tweak to not show the pop-up if it doesn't have any additional content.

**Comment by jzohrab** [comment]
Changed in the `develop` branch, will be launched with next release.

I did end up implementing the pop-up only showing when extra info is defined.  If I click "set all to well-known", then all of the terms affected will have mostly-empty data in the db, so showing a popup with just the term itself didn't feel right.  If the term has extra data, a popup is shown.

Mind you, if someone always needs to see a pop-up for a term, then we could argue that the term isn't really "well-known" :-) but that's a different story.  Maybe Lute needs a reading mode like "don't show me any highlights/colors, but still have the popups available." :-)  That's a different feature though.
